matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/membership/RoomMemberEventHandler.kt

```diff
@@ -58,10 +57,6 @@ internal class RoomMemberEventHandler @Inject constructor(
                 // but we want to preserve presence record value and not replace it with null
                 getExistingPresenceState(realm, roomId, userId))
         realm.insertOrUpdate(roomMemberEntity)
-        if (roomMember.membership.isActive()) {
-            val userEntity = UserEntityFactory.create(userId, roomMember)
-            realm.insertOrUpdate(userEntity)
-        }
```

Comment:
@ganfra was there any purpose to this being here?

Reply:
Yes, this is the way to have local User created from RoomMember state event. If you remove this, we'll loose the avatar/display name of lots of users (which is the same than a RoomMember but outside the context of a room.)
If we remove this, we should take care of fetching profile of users everywhere, which is not that great...

Reply:
Ahh I see now